invitation-frontend/src/component/Visual3.tsx

```diff
@@ -1,5 +1,723 @@
+                  <td>
+                    <span style={{ color: 'rgb(0,0,0)' }}>18</span>
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    <span style={{ color: 'rgb(180, 152, 133)' }}>19</span>
+                  </td>
+                  <td>
+                    <span>20</span>
+                  </td>
+                  <td>
+                    <span>21</span>
+                  </td>
+                  <td>
+                    <span>22</span>
+                  </td>
+                  <td>
+                    <span>23</span>
+                  </td>
+                  <td>
+                    <span>24</span>
+                  </td>
+                  <td>
+                    <span style={{ color: 'rgb(0, 0, 0)' }}>25</span>
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    <span style={{ color: 'rgb(180, 152, 133)' }}>26</span>
+                  </td>
+                  <td>
+                    <span>27</span>
+                  </td>
+                  <td>
+                    <span className={classes.current}>28</span>
+                  </td>
+                  <td>
+                    <span>29</span>
+                  </td>
+                  <td>
+                    <span>30</span>
+                  </td>
+                  <td>
+                    <span>31</span>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </Container>
+          <Container size={RESPONSIVE_MOBILE ? 550 : 450} py={85}>
+            <div className={classes.sectionTitWrap}>
+              {/* TODO: need to adjust padding with contact list */}
+              <Text
+                className={classes.sectionSubTit}
+                color="rgb(180, 152, 133)"
+                align="left"
+                size={RESPONSIVE_MOBILE ? 20 : 30}
+              >
+                Contact Us
+              </Text>
+            </div>
+            <ul>
+              <li className={classes.contactList}>
+                <div className={classes.contactListLeft}>
+                  <figure>
+                    <Image
+                      src={imgBride}
+                      alt="brideImg"
+                      width={80}
+                      height={80}
+                    />
+                  </figure>
+                  <ul style={{ marginLeft: 15 }}>
+                    <li>
+                      <Text weight={'bold'}>Call To Bride</Text>
+                    </li>
+                    <li>
+                      <Text
+                        color="#888"
+                        style={{ textDecoration: 'underline' }}
+                      >
+                        Check Her Phone number
+                      </Text>
+                    </li>
+                  </ul>
+                </div>
+                <ul style={{ display: 'flex' }}>
```

Comment:
This is as well. I think we should create new styles for each section.
ex)`contact-info-section-styles`